### ISSUE TITLE
Add wide sacroiliac joint

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10534]
**Preferred term label:**
Wide sacroiliac joint

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
The joint between the sacrum and the ilium bones of the pelvis is wider than expected

**Parent term (use hpo.jax.org/app)**
Abnormal sacroiliac joint morphology HP:0100781

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cleidocranial dysplasia (OMIM 119600), PMC6611735

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000875
- label: Wide sacroiliac joint
- parents: [Abnormal sacroiliac joint morphology (HP:0100781)](https://hpo.jax.org/app/browse/term/HP:0100781)
- pmid: 
- def: Distance between the sacrum and the ilium bones of the pelvis at the sacroiliac joint is above the upper limit of normal.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10534
- synonyms:
